mmv1/products/discoveryengine/CmekConfig.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'CmekConfig'
+description: |
+  CmekConfig represents configurations used to enable CMEK data encryption with
+  Cloud KMS keys.
+references:
+  guides:
+  api: 'https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/CmekConfig'
```

Comment:
got 404 for the link, assume the doc page is not available yet?

Maybe should be https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/projects.locations.cmekConfigs?